src/gradle/libs.versions.toml

```diff
@@ -0,0 +1,112 @@
+[versions]
+# WARNING:
+# Dependabot does not properly handle version references in build files
+# for example 'implementation "io.dropwizard.metrics:metrics-core:${libs.versions.metrics.get()}"' will be ignored/won't get PR with version update if needed
+# In some cases referincing library directly like 'libs.protobuf.protoc' may not work, possible workaround could be:
+# protoc { artifact = libs.protobuf.protoc.get().toString() } where just using 'artifact = libs.protobuf.protoc' causes error
```

Comment:
I'm sorry but I'm confused by this comment. 
Is it a `warning`, or the reason why we are migrating to version catalog?

Reply:
no, just to notify some another dev that if he decides to have in build.gradle 
```
dependencies {
  implementation org.mapstruct:mapstruct:{libs.versions.mapstruc.get()t}
}
```
instead of something like this: 
```
dependencies {
  implementation libs.mapstruct
}
```
then dependabot won't notify or create PR for new library version.
for dependencies there is no need to reference just version but in some cases it's not possible to just use `libs.mapstruct` so you may need to do some additional steps to get it working